test/thermodynamics/inner_energy_from_temperature_test.py

```diff
@@ -0,0 +1,49 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.laws.thermodynamics import inner_energy_from_temperature as inner_energy_law
+
+# Description
+## With help of calculator on https://www.calculatoratoz.com/ru/molar-internal-energy-of-ideal-gas-calculator/Calc-1705 I calculated inner energy of 1 mole (4 gram) of Helium with it's molar mass 4 gramm/mole at 20 Centigrade.
+## It should be 3656 Joules of energy.
+
+@fixture
+def test_args():
+    m = Quantity(4 * units.gram)
+    T = Quantity(295 * units.kelvin)
```

Comment:
We have a better way for degrees conversion: https://github.com/blackyblack/symplyphysics/blob/ba0753458b6c31b06351fc9959c9021ca7e8dc4d/examples/electricity/amount_energy_from_voltage_time_resistance.py#L16